rskj-core/src/test/java/co/rsk/peg/BridgeTestPowerMock.java

```diff
@@ -36,6 +36,7 @@
 import co.rsk.db.MutableTrieImpl;
 import co.rsk.peg.bitcoin.MerkleBranch;
 import co.rsk.peg.bitcoin.SimpleBtcTransaction;
+import co.rsk.peg.utils.BridgeEventLogger;
```

Comment:
I guess the QG will reject this, but just in case, please remove these unused imports